src/test/kotlin/net/atos/zac/audit/model/AuditZaakobjectWijzigingConverterTest.kt

```diff
@@ -0,0 +1,95 @@
+package net.atos.zac.audit.model
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import net.atos.client.zgw.shared.model.audit.zaken.objecten.ZaakobjectProductaanvraagWijziging
+import net.atos.client.zgw.shared.model.audit.zaken.objecten.ZaakobjectWoonplaatsWijziging
+import net.atos.client.zgw.zrc.model.Objecttype
+import net.atos.client.zgw.zrc.model.zaakobjecten.ObjectWoonplaats
+import net.atos.client.zgw.zrc.model.zaakobjecten.ZaakobjectProductaanvraag
+import net.atos.client.zgw.zrc.model.zaakobjecten.ZaakobjectWoonplaats
+import net.atos.zac.app.audit.converter.zaken.AuditZaakobjectWijzigingConverter
+import java.net.URI
+
+class AuditZaakobjectWijzigingConverterTest : BehaviorSpec({
+    val converter = AuditZaakobjectWijzigingConverter()
+
+    Given("A ZaakobjectProductaanvraagWijziging") {
+        val wijzigingen = ZaakobjectProductaanvraagWijziging()
+        val aanvraag = ZaakobjectProductaanvraag()
+        wijzigingen.nieuw = aanvraag
+        aanvraag.objectType = Objecttype.OVERIGE
+        aanvraag.`object` = URI("www.google.nl/12345")
```

Comment:
This is not a valid URI right because it does not start with `http(s)`? Our convention is to start all test URIs with http://example.com. The `example.com` domain is meant for this so I think it is a good convention.

Reply:
Good catch, thanks, I changed it